new-filename.js

```diff
@@ -0,0 +1,3 @@
+if(foo === "bar") {
```

Comment:
Codacy has a fix for the issue: [Insert `·`](https://app.dev.codacy.org/gh/troubleshoot-codacy/no-suggestions/pullRequest?prid=30887)
```suggestion
if (foo === "bar") {
```